### DISCUSSION TITLE
Question about updating

### DISCUSSION THREAD

**Discussion by barrywass** [hacksider/Deep-Live-Cam#531] [Q&A]
So I guess I installed everything about a month ago I am running a 4070 and I guess I have everything working as good as most. I know how to use GIT to pull an update from the repository but are there additional things we should be updating as well like those two additional model files. I'm just wondering will this ever get better? I know hack is using some other software made by other people if I understand it correctly and he kind of put everything together. Guess being redundant my question really is will this get better and how do we keep on top of it? Honestly for me the Snapchat face Swapper filter using a single picture is so much better and it has been around for forever so why does it feel like this is going backwards and so much more complicated? somebody anybody help

**Comment by hacksider** [comment]
The purpose of this repo is to be open-sourced and let anyone have access on it and also to learn from it. As for the snapchat face swap, this repo is not in anyway competing with it. Feel free to use any tool that you want that will serve your purpose :)

**Comment by barrywass** [comment]
Sorry I didn't mean to sound ungrateful yet I did I just reread it. Forgive me that's not what I was trying to say. But thank you for the reply and I have figured out how to update it and it is definitely getting tighter
